README.md

```diff
@@ -1,2 +1,33 @@
-# microverse-capstone-1
- HTML/CSS & JavaScript capstone project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse Capstone Project - 1 - AfriCities Summit
+
+ A website showcasing the Africities Summit in Kisumu County, Kenya
+
+## Built With
+
+- Major languages - HTML & CSS
+- Preprocessor - SASS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://walterokumu.github.io/microverse-capstone-1/)
+
+## Authors
+
+👤 **Walter Okumu**
+
+- GitHub: [@WalterOkumu](https://github.com/WalterOkumu)
+- Twitter: [@OkumuOriaro](https://twitter.com/OkumuOriaro)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```

Comment:
- [ ] Your  README should contain an "Acknowledgements" section or similar that gives appropriate credit to Cindy Shin] the author of the original design.